### PR TITLE
Adding StorSimple1200Series Azure SDK for Net

### PR DESCRIPTION
**Pull request by johnemmanuelm**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/StorSimple/Series1200/Management.StorSimple1200Series/generate.cmd [hunk 1]
```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd StorSimple1200Series/resource-manager %*
```
Comment (dsgouda) [inline src/SDKs/StorSimple/Series1200/Management.StorSimple1200Series/generate.cmd]: Please replace this with a `generate.ps1` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please run `msbuild build.proj /p:Scope=StorSimple/Series1200` and commit all the changes generated

File: src/SDKs/StorSimple/Series1200/StorSimple1200Series.Tests/StorSimple1200Series.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>StorSimple1200Series.Tests</PackageId>
+    <Description>StorSimple1200Series.Tests class library</Description>
+    <AssemblyName>StorSimple1200Series.Tests</AssemblyName>
```
[the remark this answers is not in the document]
Comment (johnemmanuelm) [reply src/SDKs/StorSimple/Series1200/StorSimple1200Series.Tests/StorSimple1200Series.Tests.csproj]: Sorry - didn't get this comment

[the remark this answers is not in the document]
Comment (dsgouda) [reply src/SDKs/StorSimple/Series1200/StorSimple1200Series.Tests/StorSimple1200Series.Tests.csproj]: Disregard this comment

**Comment by dsgouda** [comment]
@johnemmanuelm  was the code regenerated? There should be a .txt and SdkInfo_*.cs files generated which needs to be checked in

**Comment by johnemmanuelm** [comment]
Hello Deepak, As advised re-generated the client and now those two files are generated. Thanks for the help!

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please squash commits into a single commit and update the PR
Please confirm no changes have been made to the autogenerated code under Generated directory
Should be good to merge once commits are squashed

**Comment by johnemmanuelm** [comment]
No code under the Generated folder was updated. In fact had to update the rest API spec to generate it again to a pass a certain test case. https://github.com/Azure/azure-rest-api-specs/pull/3951

**Comment by dsgouda** [comment]
Clsoing in favor of https://github.com/Azure/azure-sdk-for-net/pull/4781

**Pull request closed without merging**